ui/src/components/Collection/CollectionInfo.jsx

```diff
@@ -57,38 +57,38 @@ class CollectionInfo extends Component {
   }
 
   render() {
-    const {collection, showToolbar } = this.props;
+    const {collection, showToolbar} = this.props;
     const {permissions, activeTabId, collectionXRefTab, collectionInfoIsOpen, accessIsOpen} = this.state;
-    
+
     // @TODO Discussion: 'Search Collection' link to update the current query?
     return (
       <DualPane.InfoPane className="CollectionInfo with-heading">
         {showToolbar && (
           <Toolbar className="toolbar-preview">
-            <Link to={`/search?filter:collection_id=${collection.id}`} className="pt-button button-link">
+            <Link to={`/search?filter:collection_id=${collection.id}`} className="pt-button button-link button-hover">
```

Comment:
Are you sure we need the "button-hover" class? Can this not be done within the stylesheet for the toolbar that exists now? "button-hover" strikes me as smelly code because it's basically a CSS instruction. Let's try and move it into the toolbar style :)